ui/selectors/multichain.test.ts

```diff
@@ -0,0 +1,212 @@
+import {
+  getNativeCurrency,
+  getProviderConfig,
+} from '../ducks/metamask/metamask';
+import {
+  MULTICHAIN_PROVIDER_CONFIGS,
+  MultichainNetworks,
+  MultichainProviderConfig,
+} from '../../shared/constants/multichain/networks';
+import { AccountsState } from './accounts';
+import {
+  getMultichainCurrentCurrency,
+  getMultichainDefaultToken,
+  getMultichainIsEvm,
+  getMultichainNativeCurrency,
+  getMultichainNetwork,
+  getMultichainNetworkProviders,
+  getMultichainProviderConfig,
+  getMultichainShouldShowFiat,
+} from './multichain';
+import {
```

Comment:
We should move the mocks into another constants/mocks file. Keeping it here would cause the accounts test to run whenever this test is run.

Reply:
Sounds good! Let me check how I can do that.